5.2.cup_of_join.py

```diff
@@ -0,0 +1,10 @@
+def cup_of_join_5_2(*parameters, sep='-'):
+    if not parameters:
+        return None
+    res = []
+    for par in parameters:
```

Comment:
https://workat.tech/machine-coding/tutorial/writing-meaningful-variable-names-clean-code-za4m83tiesy0 - variable names, abbreviation reference is missing